options.go

```diff
@@ -0,0 +1,141 @@
+package configparser
+
+import (
+	"strings"
+
+	"github.com/bigkevmcd/go-configparser/chainmap"
+)
+
+const defaultSectionName = "DEFAULT"
+
+// options allows to control parser behavior.
+type options struct {
+	interpolation         Interpolator
+	commentPrefixes       Prefixes
+	inlineCommentPrefixes Prefixes
+	defaultSection        string
+	delimiters            string
+	converters            Converter
+	allowNoValue          bool
+	strict                bool
+
+	// TODO: under investigation, have no effect now.
+	emptyLines bool
```

Comment:
@bigkevmcd I was thinking recently about this option, and actually I think it can be implemented, but in this case we will need to change the basics of the file parser. Since it's related to #14 issue, the process of parsing the multiline values should be added first. In this case, I would like to leave it as it is right now and come back to it after those changes will be reviewed

Reply:
I will make time over the weekend to review this and move it forward

Reply:
I added multiline value support and realisation of the empty lines inside it